src/Workspaces/Core/Portable/TaskList/TaskListItemDescriptor.cs

```diff
@@ -34,8 +41,15 @@ public static ImmutableArray<TaskListItemDescriptor> Parse(ImmutableArray<string
             {
                 if (item.Split(':') is [var token, var priorityString] &&
                     !string.IsNullOrWhiteSpace(token) &&
-                    int.TryParse(priorityString, NumberStyles.None, CultureInfo.InvariantCulture, out var priority))
+                    int.TryParse(priorityString, NumberStyles.None, CultureInfo.InvariantCulture, out var integer))
                 {
+                    var priority = integer switch
+                    {
+                        1 => TaskListItemPriority.Low,
+                        2 => TaskListItemPriority.Medium,
+                        3 => TaskListItemPriority.High,
```

Comment:
Why is this one 1-3 instead of 0-2?

Reply:
it's just how the editor stores and sends this stuff to us.  @olegtk are you aware of any reason for that?  or it just just purely historical?

Reply:
not sure. which editor component you are getting this stuff from?

Reply:
@olegtk This is the value read out of the roaming-storage location: `Microsoft.VisualStudio.ErrorListPkg.Shims.TaskListOptions.CommentTokens`.

Reply:
That's interesting, I think on VSMac we've been sending 0 for low and 2 for high...

Reply:
😢 .  oh no...

Reply:
@sandyarmstrong great!

Reply:
LOL, I don't think anybody remembers why:
```
// I've no idea why the strange conversion (legacy mapping from __VSERRORCATEGORY?).
private static int EncodedValueFromPriority(CommentTaskPriority p) { return 3 - (int)p; }
```

https://devdiv.visualstudio.com/DevDiv/_git/VS?path=/src/env/ErrorList/Pkg/Shims/TaskListOptions.cs&version=GBmain&line=133&lineEnd=134&lineStartColumn=1&lineEndColumn=98&lineStyle=plain&_a=contents

Reply:
no worries.  it is what it is :)  

I'll add a comment to this effect.  :)